batch/batch/worker/worker.py

```diff
@@ -1841,13 +1841,15 @@ async def run(self):
                             os.makedirs(self.cloudfuse_data_path(bucket), exist_ok=True)
                             os.makedirs(self.cloudfuse_tmp_path(bucket), exist_ok=True)
 
-                            await CLOUD_WORKER_API.mount_cloudfuse(
-                                credentials_path,
-                                self.cloudfuse_data_path(bucket),
-                                self.cloudfuse_tmp_path(bucket),
-                                config,
-                            )
-                            config['mounted'] = True
+                            try:
+                                await CLOUD_WORKER_API.mount_cloudfuse(
+                                    credentials_path,
+                                    self.cloudfuse_data_path(bucket),
+                                    self.cloudfuse_tmp_path(bucket),
+                                    config,
+                                )
+                            finally:
+                                config['mounted'] = True
```

Comment:
I think this makes things more confusing. I don't know what "partially mounted" means, and I would prefer to treat `config['mounted'] = True` to mean "we believe it was mounted" and not "we believe it might have been mounted". What happens if the mount failed and then we later try to unmount?